data.sql

```diff
@@ -10,3 +10,18 @@ VALUES
   ('Gabumon', '2018-11-15', 2, true, 8),
   ('Pikachu', '2021-01-07', 1, false, 15.04),
   ('Devimon', '2017-05-12', 5, true, 11);
+
+
+/* Add species column to the animals table */
+ALTER TABLE ANIMALS ADD SPECIES VARCHAR(50);
```

Comment:
* Kindly note that Everything that creates or changes the table structure should be saved in the schema.sql file. [check requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Add%20a%20column%20species%20of%20type%20string%20to%20your%20animals%20table.%20Modify%20your%20schema.sql%20file.)

Reply:
Done.